distribution/docs/src/main/resources/content/_reference/_dependencyList/ddf-dependency-list.adoc

```diff
@@ -6,358 +6,4 @@
 :summary: ${ddf-branding} ${project.version} Dependency List.
```

Comment:
✏️ Should this be `Direct Dependency List` to make it clear this does not include transitive dependencies?